interface/reports/dictation_review.php

```diff
@@ -0,0 +1,316 @@
+<?php
+/**
+ * Review Dictations
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+use OpenEMR\Core\Header;
+
+require_once("../globals.php");
+require_once("$srcdir/patient.inc");
+require_once("$srcdir/formatting.inc.php");
```

Comment:
remove above line. this is already included in the autoloader (ie. all script that include globals.php include it):
https://github.com/openemr/openemr/blob/master/composer.json#L59